Home.md

```diff
@@ -0,0 +1,141 @@
+Welcome to the ani-cli wiki!
+
+Ani-cli is a cli tool that scrapes Gogoanime and play's the anime locally on MPV(default) or VLC.
+
+# Table of Contents
+- [Flags](#Flags)
+- [Examples](#Examples)
+- [History](#History)
+- [Hacking](#Hacking)
+- [Issues](#Reporting-an-issue)
+- [contribute](#Developer-contribution)
+- [Dependencies](#Dependencies)
+- [Install](#Installation)
+  - [Arch](#Arch)
+  - [Linux](#Linux)
+  - [Mac](#Mac)
+  - [Windows](#Windows)
+  - [Android](#Android)
+- [Uninstall](#Uninstall)
+
+## Flags
+-c continue watching anime from history\
+-a specify episode to watch\
+-h show helptext\
+-d download episode\
+-p download episode to specified directory\
+-q set video quality (best|worst|360|480|720|1080)\
+-v use VLC as the media player\
+-D delete history\
+-U fetch update from github\
+-V print version number and exit
+
+## examples
+`ani-cli` starts interactive mode.\
+`ani-cli -v` starts interactive mode with vlc as media player.\
+`ani-cli -a 300 bleach` starts episode 300 of bleach.\
+`ani-cli -a 300 -v -q 720 bleach` starts episode 300 of bleach in vlc at 720p.
+
+## History
+The list of anime you have watched saves in `XDG_CACHE_HOME` or `$HOME/.cache/ani-hist`\
+the list is saved like in the forman "anime episode" for example `bleach 200` each episode is on a new line.\
+to add anime to the list eather just watch or append to the list manually with a text editor.
+
+## Hacking
+To set the defaults of the application there are some variables That can be changed under the `# default options` comment in `ani-cli` script.
+1. player_fn this can be changed to your media player of choice (VLC, or mpv(Defalt))
```

Comment:
1. player_fn this can be changed to your media player of choice (VLC, or MPV (default)) ; your media player of choice needs have a referrer title option, as the argument ```--referrer=https://gogoanime.film``` is necessary for functioning (else the video simply won't play)